data/strings/types_strings.txt

```diff
@@ -6573,6 +6613,46 @@
     zh-Hans = 园艺工
     zh-Hant = 園藝師
 
+  [type.craft.grinding_mill]
+    en = Grinding Mill
+    ar = طاحونة
+    be = Млын
+    bg = Мелница за смилане
+    ca = Molí de mòlta
+    cs = Mlýn na mletí
+    da = Slibemølle
+    de = Schleifmühle
```

Comment:
```suggestion
    de = Mühle
```

Reply:
A "Schleifmühle" is something else, it's a place where tools ar sharpende. Corn is grinded in a "Mühle".

Reply:
Fixed. Thanks!